go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/rancher
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
```

Comment:
I've seen this before when updating `go.mod`, but I think this can be removed and still work. Can you confirm?

Reply:
this is from upstream and I was trying to fix it locally but to no avail. I'll rebase when this is fixed and it should go away